assets/js/components/data-block.js

```diff
@@ -74,12 +75,11 @@ class DataBlock extends Component {
 
 		return (
 			<div
-				className={ `
-					googlesitekit-data-block
-					googlesitekit-data-block--${ context }
-					${ selected ? 'googlesitekit-data-block--selected' : '' }
-					${ className }
-				` }
+				className={ classnames(
+					`googlesitekit-data-block googlesitekit-data-block--${ context }`,
+					{ 'googlesitekit-data-block--selected': selected },
+					className
```

Comment:
I agree. I've however noticed the specificity matters. So I've just sticked with the same class order.

Reply:
Hmm, if the _order_ of these class names affects visual output I'd say that's a bug or at least a code quality issue worth fixing, because it will easily lead to bugs in the future.

That said, I don't see why the specificity would matter for https://github.com/google/site-kit-wp/blob/5a2e1e020129a2614b2308b317b26c51ff9b03e2/assets/sass/components/global/_googlesitekit-data-block.scss#L64, the only place I see `.googlesitekit-data-block--selected` used.

But if the order here matters we should clean that up. If it's straightforward it's worth doing here, but if not we should note it here in a comment and file a new issue—that's a footgun waiting to happen when someone else changes the order around 😅